src/material/dialog/dialog-container.ts

```diff
@@ -246,18 +246,18 @@ export class MatDialogContainer extends _MatDialogContainerBase implements OnDes
    * called by the dialog ref.
    */
   _startExitAnimation(): void {
-    this._animationStateChanged.emit({state: 'closing', totalTime: this._closeAnimationDuration});
+    this._animationStateChanged.emit({state: 'closing', totalTime: this._exitAnimationDuration});
     this._hostElement.classList.remove(OPEN_CLASS);
 
     if (this._animationsEnabled) {
       this._hostElement.style.setProperty(
         TRANSITION_DURATION_PROPERTY,
-        `${this._openAnimationDuration}ms`,
+        `${this._exitAnimationDuration}ms`,
```

Comment:
This is the actual fix. The rest of the changes are to align the internal terminology with the public API.